src/main/java/duke/DataFile.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class DataFile {
```

Comment:
Consider adding JavaDocs for the DataFile class and its public methods so that others can understand the methods more easily.